### ISSUE TITLE
Crash when drag card

### ISSUE THREAD

**Issue by RoshaniDias** [tispr/tispr-card-stack#22]
Hello,
When I keep dragging the card here and there without moving it down, app crash in this line
`collectionView?.bringSubview(toFront: cell!)`

in this method

```
//Define what card should we drag
    fileprivate func findDraggingCellByCoordinate(_ touchCoordinate: CGPoint) {
        if let indexPath = collectionView?.indexPathForItem(at: touchCoordinate) {
            if indexPath.item >= index {
                //top stack
                draggedCellPath = IndexPath(item: index, section: 0)
                initialCellCenter = collectionView?.cellForItem(at: draggedCellPath!)?.center
            } else {
                //bottomStack
                if (index > 0 ) {
                    draggedCellPath = IndexPath(item: index - 1, section: 0)
                }
                initialCellCenter = collectionView?.cellForItem(at: draggedCellPath!)?.center
            }
            
            //workaround for fix issue with zIndex
            let cell = collectionView!.cellForItem(at: draggedCellPath!)
            collectionView?.bringSubview(toFront: cell!)
            
        }
    }
```

How to overcome this issue. Please help me.

**Comment by bannn-1018** [comment]
you can unwrap like this:
`if let draggedCellPath = draggedCellPath, let cell = collectionView?.cellForItem(at: draggedCellPath) {
                collectionView?.bringSubview(toFront: cell)
            }`

**Comment by adudenamedruby** [comment]
This does not actually fix the issue. It causes weird behaviour with the index such that cards start appearing on top of others and cards simply disappear as well. I will look more into it to see if I can find someway to fix the crash but first I have to go through and understand the code.

**Comment by Pitsko** [comment]
@RoshaniDias Fix is the next
just add if to check if index > 0
in method
 internal func handleSwipe(_ sender: UISwipeGestureRecognizer) {
        switch sender.direction {
        case UISwipeGestureRecognizerDirection.up:
            if index > 0 {
                index -= 1
            }
}}

I will publish new version as soon as I can

**Comment by Pitsko** [comment]
Fixed in 1.0.1
